keyboards/v60_type_r/v60_type_r.h

```diff
@@ -105,3 +105,33 @@ enum my_keycodes {
       { k40, k41, k42, k43,   k44, k45,   k46, k47 } \
   }
 
+
+/* LAYOUT_iso
+ * ,-----------------------------------------------------------------------------------------.
+ * | K00 | K01 | K02 | K03 | K04 | K05 | K06 | K07 | K08 | K09 | K0A | K0B | K0C | K0D       |
+ * |-----------------------------------------------------------------------------------------+
+ * | K10    | K11 | K12 | K13 | K14 | K15 | K16 | K17 | K18 | K19 | K1A | K1B | K1C | K1D    |
+ * |---------------------------------------------------------------------------------        |
+ * | K20     | K21 | K22 | K23 | K24 | K25 | K26 | K27 | K28 | K29 | K2A | K2B | K2C |       |
+ * |-----------------------------------------------------------------------------------------+
+ * | K30 | K31 | K32 | K33 | K34 | K35 | K36 | K37 | K38 | K39 | K3A | K3B | K3C             |
+ * |-----------------------------------------------------------------------------------------+
+ * | K40   | K41   | K42   | K43                             | K44   | K45   | K46   | K47   |
+ * `-----------------------------------------------------------------------------------------'
+ */
+#define LAYOUT_60_iso( \
+	       k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, \
+	       k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, \
+	       k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, \
```

Comment:
```suggestion
	       k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k1d, \
```

Notice the same identifier deleted from the line above is added to this line, but the electrical matrix definition (Lines 129-136) is unchanged.